### PR TITLE
ipu: virtio-ipu4 as default IPU DM

### PR DESCRIPTION
**Pull request by shiqingg**
Signed-off-by: Yew, Chang Ching <chang.ching.yew@intel.com>
Reviewed-by: Edwin Zhai <edwin.zhai@intel.com>
Reviewed-by: Minggui Cao <minggui.cao@intel.com>
Acked-by: Eddie Dong <eddie.dong@intel.com>

### CODE REVIEW

File: devicemodel/samples/apl-mrb/launch_uos.sh [hunk 1]
```diff
@@ -210,22 +216,26 @@ echo "0000:00:18.0" > /sys/bus/pci/devices/0000:00:18.0/driver/unbind
...
+    # please use virtual slot 22 for i2c 0:16.0 to make sure that the i2c controller
+    # could get the same virtaul BDF as physical BDF
+    if [ -d "/sys/bus/pci/devices/0000:00:16.0" ]; then
+        echo "8086 5aac" > /sys/bus/pci/drivers/pci-stub/new_id
+        echo "0000:00:16.0" > /sys/bus/pci/devices/0000:00:16.0/driver/unbind
+        echo "0000:00:16.0" > /sys/bus/pci/drivers/pci-stub/bind
+        boot_ipu_option="$boot_ipu_option"" -s 22,passthru,0/16/0 "
+    fi
+else
+    boot_ipu_option="$boot_ipu_option"" -s 21,virtio-ipu4 "
```
Comment (kushal179) [inline devicemodel/samples/apl-mrb/launch_uos.sh]: virtio-ipu4 needs to be changed to virtio-ipu. IPU DM that has been merged expect the device name to be virtio-ipu
'-s 21,virtio-ipu'
DM patch github PR: https://github.com/projectacrn/acrn-hypervisor/pull/723. 
Please make this change

**Comment by shiqingg** [comment]
Patch has been updated per Kushal's request.

**Comment by lijinxia** [comment]
Launch UOS check failed! The patch should be dependent on some kernel patches.

**Comment by NanlinXie** [comment]
Verified after rebase PKT kernel.

**Pull request merged**